src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -564,11 +564,12 @@ private void markAsRead() {
         }
     }
 
-    private void createNewWatchListMenuItem() {
+    /**
+     * Creates a new watch list panel containing the currently selected issue
+     */
+    private void createNewWatchListPanel() {
         Optional<GuiElement> item = listView.getSelectedItem();
-        if (!item.isPresent()) {
-            return;
-        }
+        assert item.isPresent();
```

Comment:
Is it possible that the item is removed from the list between the right click and this method being executed? If that is possible, then it should be an exception.